### PR TITLE
Hook Execute modal to Sources

### PR DESCRIPTION
**Pull request by PanSpagetka**
Hook Execute playbook modal to Sources api.
DO NOT merge this before summit.

https://projects.engineering.redhat.com/browse/RHCLOUD-5817

### CODE REVIEW

File: src/components/ExecuteButton.js [hunk 1]
```diff
@@ -146,6 +150,7 @@ const ExecuteButton = ({
         : status;
 
     useEffect(() => {
+        console.log('prvni');
```
Comment (karelhala) [inline src/components/ExecuteButton.js:153]: Console.log first

File: src/components/ExecuteButton.js [hunk 2]
```diff
@@ -159,14 +164,18 @@ const ExecuteButton = ({
     }, [ data ]);
 
     useEffect(() => {
+        console.log('druhy');
```
Comment (karelhala) [inline src/components/ExecuteButton.js:167]: Console.log second

**Comment by ryelo** [comment]
Going to +1 Karel's comment. This may be a project that we end up handing off in the future. We should try to improve readability whenever we can.

**Comment by PanSpagetka** [comment]
Yep, I agree. It started simple, but it got a little bit out of hand...

**Pull request merged**